chia/cmds/farm_funcs.py

```diff
@@ -233,5 +237,16 @@ async def summary(rpc_port: int, wallet_rpc_port: int, harvester_rpc_port: int,
     if blockchain_state is not None and plots is not None:
         proportion = total_plot_size / blockchain_state["space"] if blockchain_state["space"] else -1
         minutes = int((await get_average_block_time(rpc_port) / 60) / proportion) if proportion else -1
-    print("Expected time to win: " + format_minutes(minutes))
-    print("Note: log into your key using 'chia wallet show' to see rewards for each key")
+
+    if plots is not None and len(plots["plots"]) == 0:
+        print("Expected time to win: Never (no plots)")
+    else:
+        print("Expected time to win: " + format_minutes(minutes))
```

Comment:
2 Weeks ago in the last PR that touched this code I noticed (and pointed out) that there's a small bug here. If the else-clause in the `proportion` definition applies, its value will be `-1` and the minutes will be negative. 

You could add a check and warning log in for that case here for the moment. 

(I think the ticket tab is used in another fashion right now and I've some other PR's out, before I do a refactoring. In fact this whole section bugs me a bit because the `proportion` is double inverted for no real gain)

Reply:
Thanks for the comments. I believe -1 minutes will print "unknown" for the expected time to win (unknown is used here for several error conditions - it will also print unknown for "plots is None"). Since there are a few possible error conditions that might result in unknown I'm not sure adding a log for one such condition is helpful? Note I was trying to make a relatively small change if it's obvious one has no plots then one can never win.
I'm happy to change to your code suggestion (I'm a somewhat more verbose coder in general)

Reply:
Certainly don't need to change it for me, but I just noted that i the code is repeated unnecessarily.`print("Expected time to win: "` etc.
(As for the minutes, I'd definately call it a bug since it's not not that `minutes` will be set to `-1`, it will compute the block time and divide it by `-1`. Let's say it's just sloppy default values.)